LabExT/Movement/Calibration.py

```diff
@@ -119,3 +206,45 @@ def connect_to_stage(self) -> bool:
             raise e
 
         return False
+
+    def fix_coordinate_system(self, axes_rotation: Type[AxesRotation]) -> bool:
+        if not axes_rotation.is_valid:
+            raise CalibrationError(
+                "The given axis assignment does not define a valid 90 degree rotation. ")
+
+        self._axes_rotation = axes_rotation
+        self._state = State.COORDINATE_SYSTEM_FIXED
+
+        return True
+
+    #
+    #   Movement Methods
+    #
+
+    def wiggle_axis(
+            self,
+            axis: Axis,
+            axes_rotation: Type[AxesRotation],
+            wiggle_distance=1e3,
+            wiggle_speed=1e3):
+        """
+        Wiggles the requested axis positioner in order to enable the user to test the correct direction and axis mapping.
+        """
+        x_stage, y_stage, z_stage = axes_rotation.chip_to_stage(np.array([
```

Comment:
Wouldn't it make sense to access `self._axes_rotation` here?

Also: wouldn't it make sense to check if the current `_state` is COORDINATE_SYSTE_FIXED? The method is currently written such that it only works in this state.

Reply:
Mhm good point. I wrote the `wiggle_axis` method this way to test a given `axes_rotation` (that's why as an argument to the method) before it is saved (i.e. `self._axes_rotation` is set.) That's why I deliberately didn't use `self._axes_rotation` and don't check `COORDINATE_SYSTEM_FIXED`.
`COORDINATE_SYSTEM_FIXED` is only set when the method `fix_coordinate_system` is called and a rotation is set for `self._axes_rotation`.

Is this too confusing? My idea was that `self._axes_rotation` is not set and used for general relative movements until you are satisfied with the rotation.

Reply:
Ahh, okay. This makes perfectly sense and I agree with that design pattern.